src/components/Match/matchColumns.jsx

```diff
@@ -838,6 +848,7 @@ export const goldReasonsColumns = [heroTdColumn].concat(Object.keys(strings).fil
   sortFn: row => (row.gold_reasons ? row.gold_reasons[gr.substring('gold_reasons_'.length)] : 0),
   displayFn: (row, column, value) => value || '-',
   relativeBars: true,
+  sumFn: row => (row.gold_reasons ? (row.gold_reasons[gr.substring('gold_reasons_'.length)] || 0) : 0),
```

Comment:
isn't this the same as sortFn?

The || 0 shouldn't be necessary I think

Reply:
Pretty much!
The`|| 0` is needed since the value could be undefined. If it is, then the `.reduce(sum, ...)` doesn't work and always returns undefined.
`sortFn` could probably use the same logic, or something similar, but it's probably not as noticeable since undefined probably sorts similarly to `0`.

Reply:
In that case maybe add the || 0 to the sortFn and then you can replace this line with `true` to reduce duplication?